src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -234,6 +234,10 @@
     attachCustomDimensionsAndMetrics(eventHitBuilder, properties);
     attachCampaignData(eventHitBuilder, track);
 
+    if (event.equals("Deep Link Opened")) {
```

Comment:
I think that our server-side and ajs integrations do case insensitive lookups of spec events. Should we do that here as well?

Reply:
@ccnixon this is autofired by our analytics android SDK under the hood as "Deep Link Opened". https://github.com/segmentio/analytics-android/blob/01d26bf09ba99d36ec851ce36c8bb39c6389de7c/analytics/src/main/java/com/segment/analytics/AnalyticsActivityLifecycleCallbacks.java#L110 So I don't think we need to change? Let me know if you disagree.